src/server/controllers/templateController.js

```diff
@@ -0,0 +1,102 @@
+const Quiz = require('../models/quiz');
+const QuizTemplate = require('../models/templates');
+
+const fetchTemplates = async (req, res) => {
+  try {
```

Comment:
Post adding the tag to a template, it would be better to return filtered response rather than the whole db